apps/pipelines/nodes/nodes.py

```diff
@@ -586,9 +618,10 @@ def _process(self, input, state: PipelineState, node_id: str, **kwargs) -> Pipel
             },
         )
 
-    def _get_assistant_runnable(self, assistant: OpenAiAssistant, session):
+    def _get_assistant_runnable(self, assistant: OpenAiAssistant, session: ExperimentSession, node_id: str):
+        history_manager = PipelineHistoryManager.for_assistant()
```

Comment:
It feels wrong, but no. Since the assistant only uses the `add_messages_to_history` method in the history manager and it only saves metadata on the manager, the `session` is not needed. If we ever wanted the assistant node to have history types, then we'd need the session